app/src/main/java/com/example/taskyy/ui/LoginScreen.kt

```diff
@@ -158,15 +173,21 @@ fun CreateHidePasswordToggle(state: LoginState, onEvent: (LoginEvent) -> Unit) {
 }
 
 @Composable
-fun CreateLoginButton() {
+fun CreateLoginButton(state: LoginState, onEvent: (LoginEvent) -> Unit, navController: NavController, text: String) {
```

Comment:
Same here, I'd only pass down to the button what it really needs. It doesn't need to know the whole login screen's UI, but rather just a text and a lambda when it's clicked. Then you can again reuse it for different purposes